CompileIO/compile.io
# 115b

# Addressed Issue(s)

#115

# Goals

Have the export grades button work

# Special Considerations

had to download another npm package, you might need to run npm install --save angular5-csv, but it is doubtful

# Acceptance Criteria

Export grades and look at the CSV file created, and make sure it is up to par

# Testing

- make sure you are a professor
- make sure there are some grades for your "students" to see full functionality
- go to your class, then a section, and click the students button
- once there you should see a Export grades button, click it
- check the csv and see if it is up to par

# Branch

115b
